server/handlers/handlers.go

```diff
@@ -350,14 +350,26 @@ func (ah *ActivityHandler) handleCreatedActivity(msg *clientmodels.Message, subs
 		return metrics.DiscardedReasonOther
 	}
 
-	post, errorFound := ah.msgToPost(channelID, senderID, msg, chat, false)
+	post, skippedFileAttachments, errorFound := ah.msgToPost(channelID, senderID, msg, chat, false)
 
 	newPost, appErr := ah.plugin.GetAPI().CreatePost(post)
 	if appErr != nil {
 		ah.plugin.GetAPI().LogWarn("Unable to create post", "error", appErr)
 		return metrics.DiscardedReasonOther
 	}
 
+	if skippedFileAttachments {
+		_, appErr := ah.plugin.GetAPI().CreatePost(&model.Post{
+			ChannelId: newPost.ChannelId,
+			UserId:    ah.plugin.GetBotUserID(),
+			Message:   "One or more attachments sent from Microsoft Teams were not delivered to Mattermost.",
```

Comment:
@lieut-data - I propose shortening to: "Attachments sent from Microsoft Teams weren't delivered to Mattermost."

Reply:
Good point, @JulienTant! Which is our intent with this communication? Are we communicating an error state, or communicating a consequence/outcome?

Reply:
My understanding of the code is: when the configuration option for file synchronization is "off", we want to inform the users that the files attached to this post are not sent to the other platform.

Reply:
Perfect. It's an outcome rather than an error. I agree with your recommendation of using present tense, @JulienTant: "Attachments sent from Microsoft Teams aren't delivered to Mattermost."